### DISCUSSION TITLE
Upgrading on Digital Ocean Droplet

### DISCUSSION THREAD

**Discussion by cfepdan** [Budibase/budibase#5276] [Help] [answered]
I have a deployment running on Digital Ocean using the Budibase Droplet in the DO Marketplace. It's running Budibase v1.0.90. When I click the "Check for updates" button, I'm told I'm using the latest version available, even though 1.0.108 is released.

Is there a setting I'd need to change to get the droplet to check Github? Or something else I need to do to get it to pull the most recent release?

**Comment by cfepdan** [comment, marked as answer]
Got it. Per #5283 watchtower container had stopped.
`docker-compose up -d watchtower-service` and `docker logs root_watchtower-service_1` were my friends.

**Reply by hendrikjap** [reply]
I had the same issue previously. Thanks for the solution.

**Reply by MrGonzo1024** [reply]
Hi, 

Thanks @hendrikjap I didn't originally find that article. 

I tried it. But now a message pops up saying "Updating Budibase", then after a second. it disappears. And then nothing. I tried restarting after, but it's still showing v1.0.90

**Reply by MrGonzo1024** [reply]
NM. Got it to work. I waited a little bit and tried it again. Looks like it worked this time. Thanks everyone!
